run/gust.node8.sh

```diff
@@ -7,17 +7,12 @@
 #PBS -A UNDM0006
 
 
-module purge
-module load nvhpc
-module load cray-mpich
-module load ncarcompilers
-module unload cuda
+module --force purge
+module load ncarenv/23.03
+module load nvhpc/23.1
+module load ncarcompilers/0.8.0
+module load cray-mpich/8.1.25
```

Comment:
For CPU run, shall we use the intel compiler instead for better performance?

Reply:
Jian, we could you if want.  While on Cheyenne the Intel compiler offers significant advantages versus nvhpc, however for the AMD processors, I don't see much difference between Intel and the nvhpc compiler.

Reply:
Oh really, I did not realize that. I am still under the impression that the intel compiler outperforms the nvhpc compiler significantly on the CPU but it looks like this is limited to the Intel's CPU only? This is important information to be recorded and tested in other models, too.

Reply:
I was actually surprised as well.  It is illustrated in Figure 5 of the CM1 paper.  I have not checked for other codes yet.